Packs/CommonScripts/Scripts/CheckContextValue/CheckContextValue.py

```diff
@@ -0,0 +1,55 @@
+import re
+from typing import Any, Dict, Tuple
+
+from CommonServerPython import *
+
+
+def check_key(field_value, regex=None):
+    if regex:
+        if re.match(regex, field_value):
+            return True
+    else:
+        if field_value:
+            return True
+    return False
+
+
+def poll_field(args: Dict[str, Any]) -> Tuple[str, dict, dict]:
+    keys_list = args.get('key', '').split(".")
+    regex = args.get('regex')
+    ignore_case = argToBoolean(args.get('ignore_case', 'False'))
+
+    regex_ignore_case_flag = re.IGNORECASE if ignore_case else 0
+    regex = re.compile(regex, regex_ignore_case_flag) if regex else None
+
+    context = dict_safe_get(demisto.context(), keys_list)
```

Comment:
better to use `demisto.dt` in this case because `dict_safe_get` will not work properly on multiple values.
example:
![image](https://user-images.githubusercontent.com/30797606/108036142-849c8480-7040-11eb-8bfb-1f3ade95412f.png)

when searching for `Content` value under the `Bar.Hello` key